app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java

```diff
@@ -27,6 +39,13 @@ public static LoginFragment newInstance() {
 
     @Override
     public void init() {
+        requireActivity().getWindow().setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_ADJUST_NOTHING);
+
+        emailField = getView().findViewById(R.id.vLoginEmailField);
+        passwordField = getView().findViewById(R.id.vLoginPasswordField);
+        loginButton = getView().findViewById(R.id.vLoginLoginButton);
+        signupButton = getView().findViewById(R.id.vLoginSignupButton);
+        termsAndConditions = getView().findViewById(R.id.vLoginTermsAndConditionsText);
```

Comment:
I think this is great but, can't you do this `getView().finidViewById(...)` when you define your `private TextView emailFiield = ...`?

Reply:
I tried that, but I think the view is null when the variables are defined, therefore throwing NullPointerException :(